### PR TITLE
improve problem test coverage

### PR DESCRIPTION
**Pull request by n0npax**

### CODE REVIEW

File: pkg/minikube/problem/problem_test.go [hunk 1]
```diff
@@ -17,10 +17,92 @@ limitations under the License.
...
+	var tests = []struct {
+		description string
+		problem     Problem
+		expected    string
+	}{
+		{
+			problem:     Problem{},
+			description: "empty",
+			expected: `
+* Error: [] <no value>
```
Comment (tstromberg) [inline pkg/minikube/problem/problem_test.go]: "empty" feels like a "change detector" test, do you mind removing it? I consider the output of an empty problem as undefined behavior.

Context: https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html

File: pkg/minikube/problem/problem_test.go [hunk 2]
```diff
@@ -17,10 +17,92 @@ limitations under the License.
...
+* Error: [example] test
+* Suggestion: 
+* Documentation: example.com
+`,
+		},
+		{
+			problem:     Problem{Issues: []int{0, 1}},
+			description: "with 2 issues",
+			expected: `
+* Error: [] <no value>
```
Comment (tstromberg) [inline pkg/minikube/problem/problem_test.go]: Could these test cases be updated to always have an error associated to them? This way we test real-world behavior, and don't codify the string output of undefined behavior.

Comment (n0npax) [reply pkg/minikube/problem/problem_test.go]: fixed

File: pkg/minikube/problem/problem_test.go (hunk 1 above)
Comment (n0npax) [reply pkg/minikube/problem/problem_test.go]: fixed.

**Comment by n0npax** [comment]
happy to help :)

**Pull request merged**